process/02-onboarding/stakeholder-presentation-en.md

```diff
@@ -0,0 +1,78 @@
+build-lists: true
+
+# Stakeholder Onboarding
+
+^ Use "Sketchnote" theme
+
+---
+
+# Agenda
+
+- The nature of software development
+- Risks in developing software
+- Practices we embrace
+
+---
+
+> Software creation is an empirical activity
+
+---
+
+# Empirical Process
+
+- Cannot define a 100% accurate specification upfront
+- Cannot reliably estimate effort and cost
+- Impossible to accurately schedule
+- Requires creative adaptation
+
+---
+
+![inline](./triangle.png)
+
+---
+
+# Risks in software development
+
+- Going over budget
+- Late delivery
+- Not meeting requirements/expectations
+  - features, performance, reliability, durability
+
+---
+
+# Practices we embrace
+
+- Dedicated Product Owner
+- Iterative approach
+- Feedback early and often
+- Executable specifications
+- Open source
```

Comment:
In the notes, we should have the benefits of these and how they help reduce the risk of the big 3 problems.